tests/test_complete_run.cfg

```diff
@@ -0,0 +1,98 @@
+[default]
+input = /lcrc/group/e3sm/ac.forsyth2/E3SM_simulations/20210122.v2_test01.piControl.ne30pg2_EC30to60E2r2-1900_ICG.chrysalis
+input_subdir = archive/atm/hist
+output = /lcrc/group/e3sm/ac.forsyth2/zppy_test_output/20210122.v2_test01.piControl.ne30pg2_EC30to60E2r2-1900_ICG.chrysalis
```

Comment:
These files are too big to store in the `zppy` repo. Should we just tell users to edit this file to set their own `output` and `www`? (That will require some modifications to `test_complete_run.py`).

Reply:
That's probably the only option.